configure

```diff
@@ -100,6 +101,12 @@ parse_opts() {
                 continue
                 ;;
 
+            --enable-clouseau-tests)
```

Comment:
Jan and I discussed elsewhere that this could be `--enable-clouseau` if someone does the work to add `boot_clouseau` to dev/run.